FlaskApp/api/routes.py

```diff
@@ -0,0 +1,198 @@
+from flask_jwt_extended import(create_access_token, create_refresh_token, jwt_required,jwt_refresh_token_required,get_jwt_identity,get_raw_jwt)
+from flask_restful import Resource, reqparse
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask import request, jsonify
+import json
+
+
+from.user import User
+from .utils import Validator
+from .rideoffers import RideOffer
+from .request import RequestsJ
+
+rideoffer = RideOffer()
+validator = Validator()
+join_req =  RequestsJ()
+
+user_r = User()
```

Comment:
This is not how `OOP` is used to create instances of `RideOffer(), Validator(), RequestsJ(), User()` we use it to create many instances of whatever we are trying to achieve but not a single instance used all over.